taskcluster/ci/generate-screenshots/kind.yml

```diff
@@ -13,14 +13,15 @@ transforms:
 kind-dependencies:
     - build
 
+not-for-locales:
+    - en-US   # Already built as part of the build task
+
+locales-per-chunk: 6
```

Comment:
Here's how I came up with this number.

Our bitrise plan have a 90-minute-limit. Thanks to @isabelrios' great work in https://github.com/mozilla-mobile/firefox-ios/pull/6420, 6 locales takes between 50 to 60 minutes to complete when everything is fine. If some tests fail, xcode will rerun the failed tests, adding more minutes to the counter.

A 7th locale put some of jobs to 70 minutes, which only gives 20 minutes in case of a problem. So I think 6 locales is a sweet spot.

We have 92 locales which yields 16 chunks. Our bitrise plan has 3 concurrent builds. So, the full suite will take about 6 hours to complete. I think it's a great improvement based on the current situation 😃

Reply:
Yeah, I think so too...the time spent is reduced by more than half of the current time needed...